packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -125,11 +125,17 @@ const helpers = {
         },
         httpContent
       );
+    } else {
+      // no httpContent found, returning empty body
+      return {
+        code,
+        mediaType: 'text/plain',
+        example: {
+          externalValue: '',
+          key: 'default',
+        },
+      };
     }
-    // a httpContent for default mediaType does not exist
-    throw new Error(
```

Comment:
I'm wondering whether this should not be controllable through flags `--strict` or something.

Reply:
SO-98 created, so this is resolved.